### ISSUE TITLE
Split queries when fail to connecto to confd give wrong error message

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#417]
Originally reported of Google Code with ID 361.

```
As follows:

$ gnt-group list
Group   Nodes Instances AllocPolicy NDParams
cold        0         0 preferred   spindle_count=1, exclusive_storage=False, oob_program=
default     0         0 preferred   spindle_count=1, exclusive_storage=False, oob_program=
$ pkill ganeti-confd
$ gnt-group list
Cannot communicate with the master daemon.
Is it running and listening for connections?

This needs to be improved and show better what daemon fails.
```
Originally added on 2013-02-06 15:26:58 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed:

$ ./scripts/gnt-group list
Cannot communicate with the config daemon.
Is it running and listening for connections?
$ ./scripts/gnt-instance list
Cannot communicate with the master daemon.
Is it running and listening for connections?

```
Originally added on 2013-02-12 13:55:15 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
